### PR TITLE
fix: Converted components to stateless components

### PR DESCRIPTION
**Pull request by hyperxtend**

### CODE REVIEW

File: src/components/board/board.jsx [hunk 1]
```diff
@@ -1,42 +1,41 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types'
+
```
Comment (strange-developer) [inline src/components/board/board.jsx]: Remove this empty line

File: src/components/board/board.jsx [hunk 2]
```diff
@@ -1,42 +1,41 @@
...
+import React from 'react';
+import PropTypes from 'prop-types';
 import { Container, Row } from 'react-bootstrap';
+
 import Square from '../square';
```
Comment (strange-developer) [inline src/components/board/board.jsx:6]: Single empty line preferred here rather than two empty lines

File: src/components/board/board.jsx [hunk 3]
```diff
@@ -1,42 +1,41 @@
...
+          {renderSquare(7, squares, onClick)}
+          {renderSquare(8, squares, onClick)}
         </Row>
-      </Container>
-    );
-
-  }
-}
-Board.propTypes = {
+      </Container>
```
Comment (strange-developer) [inline src/components/board/board.jsx]: Linting doesn't seem to be working in the app. These issues should be picked up when running the linting on your project.

File: src/components/square/square.jsx [hunk 4]
```diff
@@ -2,19 +2,17 @@ import React from 'react';
...
 
-export default function Square (props) {
-  return (
-    <Button
-    variant="secondary outline-dark"
-    className="square"
-    onClick={props.onClick}>
+ const Square = (props) => 
+
+    <Button variant="secondary outline-dark" className="square" onClick={props.onClick}>
```
Comment (strange-developer) [inline src/components/square/square.jsx]: Parentheses around your component here

**Review by strange-developer — CHANGES_REQUESTED** [review]

File: src/components/board/board.jsx (hunk 3 above)
Comment (hyperxtend) [reply src/components/board/board.jsx]: Sorted all of it out

File: src/components/square/square.jsx [hunk 5]
```diff
@@ -2,15 +2,16 @@ import React from 'react';
...
+      onClick: PropTypes.func,
+      value: PropTypes.string
+    };
 
-Square.propTypes = {
-  onClick: PropTypes.func,
-  value: PropTypes.string
-}
\ No newline at end of file
+
```
Comment (strange-developer) [inline src/components/square/square.jsx:24]: Single empty line preferred

File: src/components/square/square.jsx [hunk 6]
```diff
@@ -2,15 +2,16 @@ import React from 'react';
...
-   return(
-     <Button variant="secondary outline-dark" className="square" onClick={props.onClick}>
+ const Square = (props) => {
+    return(  
+    <Button variant="secondary outline-dark" className="square" onClick={props.onClick}>
       {props.value}
     </Button>)
-};
+    }
+    Square.propTypes = {
```
Comment (strange-developer) [inline src/components/square/square.jsx:15]: Empty line between prop types and component

**Review by strange-developer — CHANGES_REQUESTED** [review]

File: src/components/square/square.jsx (hunk 6 above)
Comment (hyperxtend) [reply src/components/square/square.jsx:15]: Added the line space between component & prop types

File: src/components/square/square.jsx (hunk 5 above)
Comment (hyperxtend) [reply src/components/square/square.jsx:24]: Removed extra line space

**Review by strange-developer — APPROVED** [review]

**Pull request merged**